homeworks/kirkovvalentiniván/7_extract_transform/cripto_investment.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import pandas as pd
+
+class CoingeckoAPI: 
+    
+    def __init__(self,vs_currency = "usd", order = "market_cap_desc", per_page = 250, page = 1):
```

Comment:
ha a base_url és a params is átadható lenne az init-nek, akkor ez a class tényleg szuperül újra felhasználható lenne akár más endpoint-okhoz is. nem kell szétszedegetni a paramétereket külön változókba, adjon át a user egy dictionary-t amiben minden paraméter benne van ami neki kell az api híváshoz. így lenne két paramétered: base_url és params.